python/dgl/nn/pytorch/explain/pgexplainer.py

```diff
@@ -277,20 +278,17 @@ def train_step_node(self, nodes, graph, feat, tmp, **kwargs):
         if isinstance(nodes, int):
             nodes = [nodes]
 
-        (
-            prob,
-            _,
-            batched_graph,
-            batched_feats,
-            inverse_indicies,
-        ) = self.explain_node(
+        prob, _, batched_graph, inverse_indices = self.explain_node(
             nodes, graph, feat, tmp=tmp, training=True, **kwargs
         )
 
+        batched_feats = torch.cat(
+            [feat[sg.ndata[NID].long()] for sg in unbatch(batched_graph)]
+        )
```

Comment:
It's fine to get `batched_feats` from `explain_node` as `unbatch` is not free. However, users may not need to access `batched_feats` they invoke `explain_node`.